src/scripts/items/getRanges.ts

```diff
@@ -0,0 +1,19 @@
+import {ModifierRange, RUNEWORDS, SET_ITEMS, UNIQUE_ITEMS} from "../../game-data";
```

Comment:
Nitpick: I'd still leave this file in the `comparison/` folder, ranges fall in that category.